### ISSUE TITLE
Unexpected token ILLEGAL

### ISSUE THREAD

**Issue by yaronyg** [jxcore/jxcore#823]
I'm getting the error below intermittently when i run JXcore v0.3.1.1. 

```
Yarons-MacBook-Pro:install yarongoland$ jx npm run setupDesktop
Preparing NPM for JXcore (v0.3.1.1) for the first run
Downloading NPM for JXcore
...
NPM for JX is ready.
executing... please wait.

> install@0.0.1 setupDesktop /Users/yarongoland/Desktop/depots/Thali_CordovaPlugin/thali/install
> ./setUpDesktop.sh

npm WARN EPACKAGEJSON thali-test-server@0.0.1 No repository field.
npm WARN EPACKAGEJSON thali-test-server@0.0.1 No license field.

> thali@2.1.0 postinstall /Users/yarongoland/Desktop/depots/Thali_CordovaPlugin/thali
> jx "/usr/local/bin/jx" npm installCordovaPlugin.js


/usr/local/bin/jx:1
 module, __filename, __dirname, setTimeout, setInterval, process) { ����
                                                                    ^
SyntaxError: Unexpected token ILLEGAL
    at Module._compile (module.js:589:25)
    at Object.Module._extensions..js (module.js:649:10)
    at Module.load (module.js:440:36)
    at Function.Module._load (module.js:405:12)
    at Function.Module.runMain (module.js:878:12)
    at startup (node.js:485:18)
    at node.js:1604:3

npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/bin/jx" "/Users/yarongoland/.jx/npm/bin/npm-cli.js" "--loglevel" "http" "--color" "true" "install" "--build-from-source"
npm ERR! node v0.10.40
npm ERR! npm  v3.3.12
npm ERR! code ELIFECYCLE
npm ERR! thali@2.1.0 postinstall: `jx "/usr/local/bin/jx" npm installCordovaPlugin.js`
npm ERR! Exit status 8
npm ERR! 
npm ERR! Failed at the thali@2.1.0 postinstall script 'jx "/usr/local/bin/jx" npm installCordovaPlugin.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the thali package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     jx "/usr/local/bin/jx" npm installCordovaPlugin.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls thali
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/yarongoland/Desktop/depots/Thali_CordovaPlugin/thali/npm-debug.log

npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/bin/jx" "/Users/yarongoland/.jx/npm/bin/npm-cli.js" "--loglevel" "http" "--color" "true" "run" "setupDesktop"
npm ERR! node v0.10.40
npm ERR! npm  v3.3.12
npm ERR! code ELIFECYCLE
npm ERR! install@0.0.1 setupDesktop: `./setUpDesktop.sh`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the install@0.0.1 setupDesktop script './setUpDesktop.sh'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the install package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ./setUpDesktop.sh
npm ERR! You can get their info via:
npm ERR!     npm owner ls install
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/yarongoland/Desktop/depots/Thali_CordovaPlugin/thali/install/npm-debug.log
```

**Comment by yaronyg** [comment]
Actually this is repo'ing for me so I'm going to assume it's my fault until proven otherwise.

**Comment by ktrzeciaknubisa** [comment]
What I saw at first, you have callled `jx jx`:

```bash
jx "/usr/local/bin/jx" npm installCordovaPlugin.js
```

**Comment by yaronyg** [comment]
Yup, I was replacing "jx npm install" with "jx install" and then later we wanted to go back so I did a global search and replace and turned "jx install" back to "jx npm install". Well this turned "jx installCordovaPlugin.js" into "jx npm installCordovaPlugin.js" and wamo we have this bug. I would have found it sooner but somehow I got a successful run between the failures and that confused me completely. But once the failures were consistent I realized it was my fault and found the problem.
